packages/compass-data-modeling/src/components/new-diagram-form.tsx

```diff
@@ -178,13 +266,19 @@ const NewDiagramForm: React.FunctionComponent<NewDiagramFormProps> = ({
         return {
           title: `Select collections for ${selectedDatabase ?? ''}`,
           description:
-            'These collections will be included to the generated diagram',
+            'These collections will be included in your generated diagram.',
           onConfirmAction: onCollectionsSelectionConfirm,
           confirmActionLabel: 'Generate',
           isConfirmDisabled:
             !selectedCollections || selectedCollections.length === 0,
           onCancelAction: onDatabaseSelectCancel,
           cancelLabel: 'Back',
+          footerText: (
+            <>
+              <strong>{selectedCollections.length}</strong>/
+              <strong>{collections.length}</strong> total collections selected.
```

Comment:
nit, the `s` here could be not plural if there is only 1 collection possible (same with the description)

Reply:
fixed in [250eb40](https://github.com/mongodb-js/compass/pull/6895/commits/250eb40823b621d83608ff7fb04de3f91f14d180)